main.go

```diff
@@ -6,52 +6,65 @@ import (
 	"path/filepath"
 	"flag"
 
-	"github.com/docker/docker/builder/dockerignore"
-	"github.com/docker/docker/pkg/fileutils"
+	"./dockerignore"
```

Comment:
dockerignore and fileutils are 2 packages that are on Github. They are actually 2 .go files. I simply put them in the local dir tree, in order to not point to the Github at all. Isn't it a mistake to point to a remote library (and not Red Hat) when building?